enarx-keep/build.rs

```diff
@@ -30,21 +33,56 @@ fn rerun_src(path: impl AsRef<Path>) -> std::io::Result<()> {
     Ok(())
 }
 
+fn create_dir(path: &PathBuf) {
+    match std::fs::create_dir(&path) {
+        Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {}
+        Err(e) => {
+            eprintln!("Can't create {:#?} : {:#?}", path, e);
+            std::process::exit(1);
+        }
+        Ok(_) => {}
+    }
+}
```

Comment:
might be sufficient to call `std::fs::create_dir_all()` instead of `create_dir()`